<launch>
  <node pkg="pointgrey_camera_driver" type="camera_node" name="barc_cam" output="screen">
    <param name="framerate" value="30"/>    
    <param name="image_transport" value="compressed" />
  </node>

  <node pkg="goldeneye" type="simple_lane_follower.py" name="simple_lane_follower" output="screen">
    <param name="display_image" value="True" type="boolean"/>
    <param name="publish_image" value="False" type="boolean"/>
    <param name="upperY" value="148" type="int"/>
    <param name="lowerY" value="65" type="int"/>
  </node>

</launch>
